src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/dec/BrotliStream.Decompress.cs

```diff
@@ -134,7 +134,8 @@ private async ValueTask<int> FinishReadAsyncMemory(Memory<byte> buffer, Cancella
                         _bufferOffset = 0;
 
                         int numRead = 0;
-                        while (_bufferCount < _buffer.Length && ((numRead = await _stream.ReadAsync(new Memory<byte>(_buffer, _bufferCount, _buffer.Length - _bufferCount)).ConfigureAwait(false)) > 0))
+                        while (_bufferCount < _buffer.Length &&
+                              ((numRead = await _stream.ReadAsync(new Memory<byte>(_buffer, _bufferCount, _buffer.Length - _bufferCount), cancellationToken).ConfigureAwait(false)) > 0))
```

Comment:
It's for cases like this that I'm glad we have the analyzer :)

Reply:
Yeah -- that's awesome -- I would never count on human eyes to have caught this one.